packages/i18n-calypso/package.json

```diff
@@ -1,48 +1,37 @@
 {
-  "name": "not-i18n-calypso-itreallyis",
-  "version": "3.0.0",
+  "name": "i18n-calypso",
+  "version": "4.0.0",
   "description": "i18n JavaScript library on top of Jed originally used in Calypso",
-  "main": "index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
```

Comment:
`sideEffects: false` should be useful here too. I don't think we have any side effects in index?

Reply:
added in https://github.com/Automattic/wp-calypso/pull/29229/commits/911996ecbcbcc655ac7743369b40b4b14c7e3fe3

Reply:
I don't think that `sideEffects` will have much effect here: all we export is parts of the `i18n` object instance. The only thing that one can shake off is the `localize` HOC. But it doesn't do any harm.